infra/notifications/src/app.js

```diff
@@ -219,6 +231,23 @@ app.delete('/mobile/register', async (req, res) => {
   }
 })
 
+/**
+ * Endpoint called by the messaging-server
+ * list of eth address that have received a message
+ */
+app.post('/messages', async (req, res) => {
+  res.status(200).send({ status: 'ok' })
+
+  const sender = req.body.sender // eth address
+  const receivers = req.body.recievers // array of eth addresses
```

Comment:
nit: what's difference between receivers and recievers ? should we use same naming ?

Reply:
The difference is my inability to spell... 😬

Reply:
Ha! Vidicated: it's actually because Yu Pan can't spell, and I blindly copied. That field comes from the messaging service, here:
https://github.com/OriginProtocol/origin/pull/2197/files#diff-8a57fbb88a1ad9dcf4f114b9e7efccb2L300
Will still fix it. :)